### ISSUE TITLE
Audio files from the original are in Nier Replicant ver.1.22?

### ISSUE THREAD

**Issue by Fulminis-ictus** [NSACloud/NieR-Audio-Tools#4]
I've extracted the files from Nier Replicant ver.1.22 using this tool to get my hands on "Ashes of Dreams" and noticed that the old versions of these songs are in the game files. I haven't yet had the chance to check if any other tracks from the original exist as well. I found it weird because there's no info about the old tracks being in the game. All I could find was people saying that the tracks from the original aren't in the remake.

Just want to make sure that the tool isn't downloading them from somewhere else for whatever reason, even though that probably doesn't make sense. The tracks in question are: 25123032, 20094670, 10518182 and 339249670. They can all be found in the sfx folder after unpacking stream2.pck.

Anyone happen to have any info on those tracks (like when they play, how they can be accessed in-game etc.)?

**Comment by NSACloud** [comment]
They're not the old versions of the songs, they're remade for the remake. Each is a different language, German, French, English, Japanese and probably something else I don't remember. I don't know when they play but you probably have to have your language set to that specific language for it to play in the credits. Automata did something similar where weight of the world is different for Japanese and English IIRC. Or they might be played for each letter ending. I haven't played it in a while.

The only music in the game that's the same as the original is the father nier mode music IIRC.

**Comment by Fulminis-ictus** [comment]
The new versions of "Ashes of Dreams" are 753268177, 640800575, 374494533 and 258587752 respectively. The ones I mentioned above are the old ones.
The new ones play during the different remake endings (the pseudo-Japanese one during ending E, the pseudo-Gaelic during ending C etc., independent of what language your game is set to). The old ones also play during the endings of the original game. Not sure where the old versions are playing in the remake, though ;_; Maybe they're in the files but aren't actually used anywhere.

Does that mean that you don't get the old versions (25123032, 20094670, 10518182 and 339249670) after extracting stream2.pck?

**Comment by NSACloud** [comment]
Yes they are there. It's possible they don't get used. There's lots of unused things left over from the original game. Sounds are less likely to contain unused things though since they had to redo the whole sound system to use wwise. So it may play in game after some condition, but what that is, I don't know.
